packages/govuk-frontend/src/govuk/components/accordion/accordion.mjs

```diff
@@ -411,7 +411,22 @@ export class Accordion extends GOVUKFrontendComponent {
     const $button = $section.querySelector(`.${this.sectionButtonClass}`)
     const $content = $section.querySelector(`.${this.sectionContentClass}`)
 
-    if (!$showHideIcon || !$showHideText || !$button || !$content) {
+    if (!$button) {
```

Comment:
Note this is our second check for `$button`, so I haven't written another test for it. I guess technically you could trigger this slightly differently from the other instance, but the end result is an error. This will be neater if we figure out some kind of caching solution.